blueprint/main.tf

```diff
@@ -111,3 +111,66 @@ resource "kubernetes_namespace" "test" {
   }
   provider = kubernetes.test
 }
+
+resource "kubernetes_deployment" "team-1-app" {
+  provider = kubernetes.dev
+
+  metadata {
+    name = "group16-team1-test-app"
+    labels = {
+      test = "MyExampleApp"
+    }
+    namespace = "namespace-${var.team_id}"
```

Comment:
I would make this a reference to the namespace, which is in this file.

Reply:
DONE. Changed to `namespace = kubernetes_namespace.dev.metadata.name`